dartagnan/src/main/java/com/dat3m/dartagnan/program/processing/CoreCodeVerification.java

```diff
@@ -30,7 +32,8 @@ public static CoreCodeVerification fromConfig(Configuration config) {
     private static final Set<Class<? extends Event>> CORE_CLASSES = new HashSet<>(Arrays.asList(
             Load.class, Store.class, Init.class, GenericMemoryEvent.class, Fence.class,
             CondJump.class, IfAsJump.class, ExecutionStatus.class, Label.class, Local.class,
-            Skip.class, Assume.class, RMWStore.class, RMWStoreExclusive.class
+            Skip.class, Assume.class, RMWStore.class, RMWStoreExclusive.class,
+            BeginAtomic.class, EndAtomic.class // We add SVCOMP atomic blocks here as well
```

Comment:
The comment is useless. It is obvious that those were added. Either explain why we need to add them or simply remove the comment